ironfish-cli/src/commands/wallet/scanning.ts

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { parseBoolean } from '../../args'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class ScanningCommand extends IronfishCommand {
+  static description = `Enable or disable scanning for an account`
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'account',
+      required: true,
+      description: 'Name of the account to update',
+    },
+    {
+      name: 'enabled',
+      parse: (input: string): Promise<boolean | null> => Promise.resolve(parseBoolean(input)),
+      required: true,
+      description: 'True if scanning should be enabled, else false',
```

Comment:
nit: `True` or `false` should we keep them the same case since this refers to a value? It doesn't matter since it lower cases it underneath but it would be nice if they refer to the same case